src/store/index.js

```diff
@@ -18,6 +18,10 @@ const initialState = {
   unconstrainedTerms: [],
   // whether the user found all the conditions they were searching for
   foundAllConditions: null,
+  // whether the user found all the conditions they were searching for
+  demographics: null,
+  // whether the user found all the conditions they were searching for
+  additionalComments: null,
```

Comment:
It looks like you duplicated the comment from `foundAllConditions` for these other properties.

Reply:
updated comment